types/audit-details.d.ts

```diff
@@ -218,12 +218,23 @@ declare global {
        */
       export interface SourceLocationValue {
         type: 'source-location';
-        /** urls from the network are always valid urls. otherwise, urls come from either a comment or header, and may not be well-formed. */
+        /** urls from the network are always valid urls. otherwise, urls come from either a comment or header (see urlProvider), and may not be well-formed. */
         url: string;
-        /** 'network' when the url is the actual, observed resource url. 'comment' when the url comes from a sourceMapURL comment or X-SourceMap header */
+        /**
+         * - `network` when the url is the actual, observed resource url.
+         * - `comment` when the url comes from a sourceMapURL comment or X-SourceMap header
```

Comment:
heh i am pretty sure you mixed yourself up.

'comment' is for a URL from `hasSourceURL`, which is [from the `// #sourceURL=..` comment. ](https://source.chromium.org/chromium/chromium/src/+/master:third_party/blink/renderer/core/inspector/inspector_style_sheet.cc;l=1725-1736;drc=7ccffaf0933ccc647c744bf66971bcf5f33a676a)

afaik there's no network header that can define a sourceurl for the resource.

Reply:
oh ya